src/views/index.tpl.html

```diff
@@ -35,6 +35,9 @@ <h1 class="site-title">
 			</li>
 		</ul>
 	</header>
+	<header class="navigator-banner-header">
+		<a href="https://cloudsmith.io/~eventstore/repos/navigator/packages/">Admin UI is legacy, and it's being replaced by EventStore Navigator. <b>Click here</b> to download EventStore Navigator</a>

```

Comment:
The link should be to the docs-next page:
```suggestion
		<a href="https://docs-next.eventstore.com/navigator/">Admin UI is legacy, and it's being replaced by EventStore Navigator. <b>Click here</b> to download EventStore Navigator</a>

```